packages/amplify-category-hosting/lib/S3AndCloudFront/helpers/file-uploader.js

```diff
@@ -71,4 +76,7 @@ async function uploadFile(s3Client, hostingBucketName, distributionDirPath, file
 
 module.exports = {
   run,
+  exportForTesting: {
+    sortUploadFiles,
+  }
```

Comment:
we should not do this. I.e. don't open internals.
instead we should mock modules that are imported into uploader and assert order of `s3Client.upload` calls.

Reply:
Done e4ab4609c43863fc57d5f4d9e431c011a3e96927